bundles/org.openhab.binding.venstarthermostat/src/main/feature/feature.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.binding.astro-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+    <repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${project.version}/xml/features</repository>
+
+    <feature name="openhab-binding-ventarthermostat" description="Venstar Thermostat Binding" version="${project.version}">
+        <feature>openhab-runtime-base</feature>
+        <bundle dependency="true">mvn:org.eclipse.jetty/jetty-client/9.4.10.v20180503</bundle>
```

Comment:
See comment about pom.xml. Should not be needed here.

Reply:
See previous comment re: jetty-client 9.4